gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Thu Feb 24 11:21:45 PST 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.6-all.zip
```

Comment:
If you use the Gradle Upgrade Manager, or the `./gradlew wrapper --gradle-version x.x.x` command, I think more files would be affected?

Reply:
@jo-elimu Updated now using the command instead.